app/javascript/components/project-submissions/components/submission.jsx

```diff
@@ -62,23 +62,25 @@ const Submission = forwardRef(({
         {isCurrentUsersSubmission
           ? (
             <div
-              className="submissions__button--edit"
+              className="submission-icon text-gray-500 hover:text-black"
               onMouseDown={toggleShowEditModal}
               role="button"
               tabIndex={0}
               aria-label="edit-button"
               data-test-id="edit-submission-btn"
-            />
+            >
+              <i className="fas fa-edit fa-lg" />
```

Comment:
With the button padding being smaller these will look slightly out of proportion, removing the fa-lg from this and the flag icon below should get them back in the right size range.